pyprophet/runner.py

```diff
@@ -95,6 +95,8 @@ def read_osw(infile, level, ipf_max_peakgroup_rank, ipf_max_peakgroup_pep, ipf_m
          FEATURE.EXP_RT ASC;
 ''', con)
             elif level == "transition":
+                if not check_sqlite_table(con, "SCORE_MS2"):
+                    raise click.ClickException("Transition-level scoring requires prior MS2 or MS1MS2-level scoring.")
```

Comment:
```suggestion
                    raise click.ClickException("Transition-level scoring for IPF requires prior MS2 or MS1MS2-level scoring. Please run 'pyprophet score input.osw' to fix this.")
```

maybe suggest how to fix this? also, this is only for IPF, right? Otherwise you would not do transition level scoring?